content/rdi/installation/pacemaker.md

```diff
@@ -343,7 +344,7 @@ Verify the container is running with `sudo podman ps -a`. This should result in
 
 ```shell
 CONTAINER ID   IMAGE             COMMAND              CREATED          STATUS          PORTS     NAMES
-2aecd4e9fa31   debezium/server   "/debezium/run.sh"   2 minutes ago    Up 2 minutes              dbz_server
+2aecd4e9fa31   quay.io/debezium/server:{{<param rdi_debezium_server_version>}}   "/debezium/run.sh"   2 minutes ago    Up 2 minutes              dbz_server
```

Comment:
This seems not indented correctly (even when replacing the `{{<param rdi_debezium_server_version>}}` with actual version. Try to replace it with 2.4.0 for example and see that it's indented as before, then put the placeholder.